Manifest.toml

```diff
@@ -0,0 +1,94 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
The Manifest.toml file probably shouldn't be included and be added to the .gitignore

Reply:
Addressed